textsubs.lic

```diff
@@ -890,4 +890,302 @@ TextSubs.add('Your skill in (.*) is at a zenith of enlightenment\.', 'Your skill
 # Commas when dealing with coins
 TextSubs.add('\d{1,3}(?=(\d{3})+(?!\d).*(Kronar|Dokora|Lirum))', '\&,')
 
+# Scroll subs to add guild name
+TextSubs.add('"Strange Arrow."', '"Strange Arrow(Analagous)."')
```

Comment:
Ok.  I saw that syntax in the spell subs part that you mentioned, but I wasn't sure exactly how it worked so I didn't try to use it.

Thanks for the explanation.